hammerdb-wrapper/hammerdb-wrapper.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
```

Comment:
The file that needs to be updated is https://github.com/cloud-bulldozer/snafu/blob/master/hammerdb/hammerd_wrapper.py, also i think you'd mentioned a need to change the dockerfile as well. Lmk once its ready :)